frontend/src/app/core/datetime/timezone.service.ts

```diff
@@ -37,13 +37,7 @@ export class TimezoneService {
   constructor(
     readonly configurationService:ConfigurationService,
     readonly I18n:I18nService,
-  ) {
-    this.setupLocale();
-  }
-
-  public setupLocale():void {
```

Comment:
This is no more required, because we set the locale in the `init-locale.ts` below.